docs/compatibility.md

```diff
@@ -23,9 +23,10 @@ versions of related services that each operator version is compatible with. The
 "Most Recent Version" column shows the last version of the operator that was
 published for each major version.
 
-| Operator Version  | Most Recent Version | Supported Cluster Models  | Supported FDB Versions  | Supported Kubernetes Versions |
-| ----------------- | ------------------- | ------------------------- | ----------------------- | ----------------------------- |
-| 0.x               | 0.51.0              | v1beta1                   | 6.1.12+                 | 1.15.0+                       |
```

Comment:
I think that we should keep this table with one entry per major version of the operator, in order to keep things simple for users of the operator. That means that we need to commit to making the changes in this PR part of a major release and retain compatibility with older versions of FDB in the 0.x family.